social/app/models/author.py

```diff
@@ -103,7 +102,14 @@ def create_profile(sender, **kwargs):
         user_profile.node = Node.objects.get(local=True)
         user_profile.save()
 
+def update_profile(sender, **kwargs):
+    from social.tasks import get_github_activity
+    author = kwargs["instance"]
+    if author.github != "":
+        time = datetime.now().replace(2018, 1, 1)
```

Comment:
Better way would be to [add a year (or more) from the current date](http://stackoverflow.com/a/15743908/2557554) but it's not necessary for our demo. I assume you specified `repeat_until` because if it is `None` it only runs once? It would be better if it didn't require an end-date expiry.